Production/python/customiseHLT.py

```diff
@@ -92,8 +92,20 @@ def customise(process):
       status = Var("status", int, doc='status word'),
       longLived = Var("longLived", bool, doc='is long lived?'),
       massConstraint = Var("massConstraint", bool, doc='do mass constraint?'),
-
-
+      # source: cmssw/PhysicsTools/NanoAOD/plugins/SimpleFlatTableProducerPlugins.cc 
+      CKFtrackDz = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dz : -999.", float, doc = "CKF track Dz"),
+      GSFtrackDz = Var("? gsfTrackRef.isNonnull && gsfTrackRef.isAvailable ? gsfTrackRef.dz : -999.", float, doc = "GSF track Dz"),
+      CKFtrackDxy = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dxy : -999.", float, doc = "CKF track Dxy"),
+      GSFtrackDxy = Var("? gsfTrackRef.isNonnull && gsfTrackRef.isAvailable ? gsfTrackRef.dxy : -999.", float, doc = "GSF track Dxy"),
+      CKFtrackIsValid = Var("trackRef.isNonnull && trackRef.isAvailable", bool, doc = "CKF track is valid"),
+      GSFtrackIsValid = Var("gsfTrackRef.isNonnull && gsfTrackRef.isAvailable", bool, doc = "GSF track is valid"),
+      
+      # CKFLostInnerHits = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.hitPattern.numberOfLostHits(reco::HitPattern::MISSING_INNER_HITS) : -999.", float, doc = "CKF track LostInnerHit"), 
+      # This doesn't work: failed to parse "? trackRef.isNonnull && trackRef.isAvailable ? trackRef.hitPattern.numberOfLostHits(reco::HitPattern::MISSING_INNER_HITS) : -999."
+      # also tried: CKFLostInnerHits = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.hitPattern.numberOfLostHits : -999.", float, doc = "CKF track LostInnerHit"), 
+      # this doesn't work: method named "numberOfLostHits" for type "reco::HitPattern" was passed the wrong number of arguments. 
```

Comment:
I think it is ok to not have these variables for the current round. But I suggest adding following variables (in addition to what you have already implemented):
```
track_dxyError
track_dzError
track_pt
track_eta
track_phi
track_ptError
track_etaError
track_phiError
track_chi2
track_ndof

track_numberOfValidHits
track_numberOfLostHits
track_missingInnerHits
track_missingOuterHits
track_validFraction
track_qualityMask

hcalEnergy
ecalEnergy
```